config_docker_certs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+sudo service docker stop
+
+sudo bash -c 'echo "[Service]
+ExecStart=
```

Comment:
remove?

Reply:
it was required to remove the existing command...